fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSExternalCatalog.java

```diff
@@ -161,25 +160,13 @@ protected synchronized void initPreExecutionAuthenticator() {
     @Override
     protected void initLocalObjectsImpl() {
         initPreExecutionAuthenticator();
-        HiveConf hiveConf = null;
-        JdbcClientConfig jdbcClientConfig = null;
-        String hiveMetastoreType = catalogProperty.getOrDefault(HMSProperties.HIVE_METASTORE_TYPE, "");
-        if (hiveMetastoreType.equalsIgnoreCase("jdbc")) {
-            jdbcClientConfig = new JdbcClientConfig();
```

Comment:
I didn’t validate whether the user input still includes any JDBC references, but I think that’s fine in this case.